TACC/tup-ui
# feat: support different buttons for TicketCreateModal

## Overview / Changes

1. Allow `TicketCreateModal` to use different size (or type) buttons.
2. Reduce size of [ + New Ticket ] on Dashboard.

## Related

- ~~required by #317~~
- superseded by #319

## Testing

1. Open Dashboard.
3. Verify "+ New Ticket" button is smaller.
4. Thus, see height of header title and action height are less different.

## UI

| before | after |
| - | - |
| ![before](https://github.com/TACC/tup-ui/assets/62723358/8602495e-f4ff-4af0-bc03-62aea815bf8a) | ![after](https://github.com/TACC/tup-ui/assets/62723358/5ef88576-2bf9-464c-b1fe-acf2eadea110) |
